Admin/AbstractAdmin.php

```diff
@@ -2804,7 +2804,7 @@ public function configureActionButtons($action, $object = null)
             && $this->hasRoute('edit')
         ) {
             $list['edit'] = array(
-                'template' => 'SonataAdminBundle:Button:edit_button.html.twig',
+                'template' => $this->getConfigurationPool()->getTemplate('button_edit'),
```

Comment:
Why we couldn't call just `$this->getTemplate('button_edit')` like in other places?